class.go

```diff
@@ -41,21 +44,29 @@ func (q HtbClassAttrs) String() string {
 // Htb class
 type HtbClass struct {
 	ClassAttrs
-	Rate    uint64
-	Ceil    uint64
+	Rate    nl.TcRateSpec
+	Ceil    nl.TcRateSpec
```

Comment:
in general i try to avoid nl specific types outside of the netlink library. It should be possible to use netlink without importing nl. If you must expose it this way, please creete a new type in class.go that mirrors the lower level nl.TcRateSpec, although i would prefer it if we could keep the types simple in netlink